.goreleaser.yml

```diff
@@ -46,6 +46,7 @@ archives:
 
 signs:
   - artifacts: all
+    args: ["-u", "0xDC9FC6B1FCE47986", "--output", "${signature}", "--detach-sign", "${artifact}"]
```

Comment:
This wasn't necessary to sign the other artifacts.  Does the yum & deb packaging make it necessary?